assets/pages/monthly-contributions/monthlyContributions.scss

```diff
@@ -1,3 +1,109 @@
 #monthly-contributions-page {
-	background-color: gu-colour(multimedia-main-2);
+
+	// ----- General ----- //
+
+	background-color: darken(gu-colour(multimedia-main-2), 5%);
+
+	// Fix for IE.
+	& > div {
+		background-color: darken(gu-colour(multimedia-main-2), 5%);
+	}
+
+	.monthly-contrib {
+		background-color: gu-colour(multimedia-main-2);
+	}
+
+	a {
+		color: gu-colour(neutral-1);
+	}
+
+
+	// ----- Sections ----- //
+
+	h2 {
+		margin-top: -2px;
+	}
+
+	// ----- Header
+
+	.monthly-contrib__header {
+		border-top: none;
+
+		.monthly-contrib__heading {
+			margin-top: -15px; /* custom size for header at mobile because of padding */
+			font-size: 24px;
+			line-height: 28px;
+			font-weight: bold;
+			font-family: $gu-egyptian-web;
+
+			@include mq($from: desktop) {
+				margin-top: $gu-v-spacing;
+				font-size: 44px;
+				line-height: 44px;
+			}
+		}
+
+		.component-secure {
+			&__text {
+				margin-top: $gu-v-spacing / 2;
+			}
+
+			// Fix for IE.
+			svg {
+				width: 13px;
+			}
+		}
+
+	}
+
+	// ----- Your contrib
+
+	.monthly-contrib__amount {
+		border-radius: 1000px;
+		font-size: 24px;
+		color: #fff;
+		width: 60px;
+		height: 60px;
+		text-align: center;
+		background-color: gu-colour(neutral-1);
+		line-height: 60px;
+	}
+
+	.monthly-contrib__amount--wide-value {
+		padding-left: 1em;
```

Comment:
In this case that the requirement is that the padding should be tied to its parent's font-size, we should use `em`.